RocketMap/RocketMap
# manual recaptcha solving

## Description

Manual Captcha Solving without 2captcha

## Motivation and Context


Someone may not have 2captcha, they need to solve the captcha themselves, it provides a easy way for them to solve the captcha

## How Has This Been Tested?



1. Go to /bookmarklet under your map.
eg: If your maps is http://xyz.com, go to http://xyz.com/bookmarklet

2. Drag the Solve Recaptcha Bookmarklet to your bookmark bar

3. Wait for an account to encounter the captcha

4. Click the bookmarklet one time, it will redirect you to http://pgorelease.nianticlabs.com/

5. Click the bookmarklet one more time, it will show the Recaptcha

<img width="170" alt="screen shot 2016-11-16 at 9 50 54 pm" src="https://cloud.githubusercontent.com/assets/918554/20349762/ceac0414-ac46-11e6-9576-cab57a4968e0.png">

5. Solve the recaptcha

6. Recaptcha token will be saved to Database in Token table

7. token_request will get a token which created after the request made and return to the search worker and verify the captcha challenge

p.s.: if token_request cannot get a token which created after the request made within manual-captcha-solving-allowance-time(default: 60s), token_request will return 'ERROR' and append the account to account_failures
Please set manual-captcha-solving-domain to your map domain eg: http://xyz.com

## Screenshots (if appropriate):

## Types of changes


## Checklist:
